pkg/promtail/scrapeconfig/scrapeconfig.go

```diff
@@ -82,7 +80,11 @@ type PushTargetConfig struct {
 
 // DefaultScrapeConfig is the default Config.
 var DefaultScrapeConfig = Config{
-	EntryParser: api.Docker,
+	PipelineStages: []interface{}{
```

Comment:
[q]: unfortunately coz the type of pipeline stages are just `[]interface{}`. Would love to know if there is a better way!